src/current/cockroachcloud/serverless-resource-usage.md

```diff
@@ -1,13 +1,19 @@
 ---
-title: Optimize Your CockroachDB Serverless Workload
-summary: Tune your workload to reduce costs
+title: Understand your Serverless Resource Usage
+summary: Diagnose your RU consumption and tune your workload to reduce costs
 toc: true
 docs_area: deploy
 ---
 
-This page describes how to tune your CockroachDB {{ site.data.products.serverless }} workload to reduce costs.
+This page describes how to diagnose your CockroachDB {{ site.data.products.serverless }} cluster's Request Unit (RU) consumption and tune your workload to reduce costs.
 
-## Understanding your resource usage
+- [Understand resource consumption in CockroachDB {{ site.data.products.serverless }}](#understand-resource-consumption-in-cockroachdb-serverless) gives an overview of the CockroachDB {{ site.data.products.serverless }} architecture that affects RU and storage consumption and explains how RUs are calculated.
+
+- [Diagnose and optimize your resource consumption](#diagnose-and-optimize-your-resource-consumption) explains how to find and optimize queries and processes that may be consuming excessive resources and provides general tips for reducing resource usage.
+
+For information on planning your cluster configuration and setting resource limits, refer to [Plan a Serverless Cluster]({% link cockroachcloud/plan-your-cluster-serverless.md %}).
+
+## Understand resource consumption in CockroachDB {{ site.data.products.serverless }}
```

Comment:
This section feels disconnected from the rest of the doc since we don't refer to it anywhere else + feels distracting from what the user actually cares about (prescriptive and actionable guidance on diagnosing their RU usage). Let me know if I'm missing historical context on why this section is here.

Reply:
Agree it's not a great fit, but it was already on this page and I think it's important context. I've been considering moving it to Plan Your Serverless Cluster but that page is also getting a bit long, especially with the multi-region additions i have to publish right after this. Do you think it should move there?

Reply:
TLDR would like to move but don't have a better place in mind

Reply:
Can we move it to the end of the doc? Let the user find the actionable guidance first and read additional material if they're interested?

Reply:
just pushed an update. what do you think of this version with some prerequisite info at the top and the example RU calculation moved under the diagnostics?

Reply:
As discussed in our 1:1, let's handle the info rearchitecture in a follow-up PR.